test/ComposerRequireCheckerTest/NodeVisitor/UsedSymbolCollectorTest.php

```diff
@@ -0,0 +1,315 @@
+<?php
+
+namespace ComposerRequireCheckerTest\NodeVisitor;
+
+use ComposerRequireChecker\NodeVisitor\UsedSymbolCollector;
+use PhpParser\Node\Expr\ClassConstFetch;
+use PhpParser\Node\Expr\ConstFetch;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Expr\Instanceof_;
+use PhpParser\Node\Expr\New_;
+use PhpParser\Node\Expr\StaticCall;
+use PhpParser\Node\Expr\StaticPropertyFetch;
+use PhpParser\Node\Expr\Variable;
+use PhpParser\Node\Name;
+use PhpParser\Node\Param;
+use PhpParser\Node\Stmt\Catch_;
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\ClassMethod;
+use PhpParser\Node\Stmt\Function_;
+use PhpParser\Node\Stmt\Interface_;
+use PhpParser\Node\Stmt\TraitUse;
+use PHPUnit\Framework\TestCase;
+
+class UsedSymbolCollectorTest extends TestCase
```

Comment:
Please add `/** @covers \ComposerRequireChecker\NodeVisitor\UsedSymbolCollector */` to prevent coverage leaks